Firestore/Example/Tests/Util/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+find_package(XCTest REQUIRED)
+
+objc_framework(
+  firebase_firestore_tests_util
```

Comment:
Objective-C only targets get an `_objc` suffix elsewhere. I think this likely makes sense here too.